checkpoints

```diff
@@ -0,0 +1 @@
+/home/PJLAB/liqikai/checkpoints/
```

Comment:
unnecessary I suppose

Reply:
You are right. It is a mistake. Will fix it.